### PR TITLE
Tutorial improvements/fixes

### PR DESCRIPTION
**Pull request by MetRonnie**
**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.

### CODE REVIEW

File: src/tutorial/runtime/runtime-configuration.rst [hunk 1]
```diff
@@ -430,19 +430,36 @@ Start, Stop, Restart
...
+      :cylc:conf:`[scheduler]allow implicit tasks`:
+
+      .. code-block:: diff
+
+          [scheduler]
+              UTC mode = True
+         -    allow implicit tasks = True  # TODO: remove at end of exercise
+
+      Removing this ensures that any tasks in the graph without a corresponding
+      runtime section will cause validation to fail (e.g. due to a typo).
```
Comment (oliver-sanders) [inline src/tutorial/runtime/runtime-configuration.rst:443]: :+1:

File: src/tutorial/runtime/runtime-configuration.rst [hunk 2]
```diff
@@ -454,10 +471,8 @@ Start, Stop, Restart
 
          Run the workflow either by:
 
-         * Pressing the play button in the Cylc GUI. Then, ensuring that
-           "Cold Start" is selected within the dialogue window, pressing the
-           "Start" button.
-         * Running the command ``cylc play runtime-tutorial``.
+         * TUI/GUI: pressing play for the latest run.
```
Comment (oliver-sanders) [inline src/tutorial/runtime/runtime-configuration.rst:474]:
```suggestion
         * GUI: pressing play for the latest run.
         * Tui: pressing enter on the workflow and selecting "play".
```

Comment (MetRonnie) [reply src/tutorial/runtime/runtime-configuration.rst:474]: The user will have already played the first run in a previous step, so I tried to keep this short. Happy to commit though if you want

Comment (oliver-sanders) [reply src/tutorial/runtime/runtime-configuration.rst:474]: That's fine, no need to repeat the same advice, maybe we could just link back to it?

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**